src/cpp/client/channel_cc.cc

```diff
@@ -71,65 +73,79 @@ class ChannelConnectivityWatcher {
     char* env = gpr_getenv("GRPC_DISABLE_CHANNEL_CONNECTIVITY_WATCHER");
     bool disabled = false;
     if (env != nullptr) {
-      static const char* truthy[] = {"yes",  "Yes",  "YES", "true",
-                                     "True", "TRUE", "1"};
+      static const char* truthy[] = {"yes", "true", "1"};
       for (size_t i = 0; i < GPR_ARRAY_SIZE(truthy); i++) {
-        if (0 == strcmp(env, truthy[i])) {
+        if (0 == gpr_stricmp(env, truthy[i])) {
           disabled = true;
           break;
         }
       }
     }
     gpr_free(env);
     if (!disabled) {
+      gpr_ref_init(&ref_, 0);
```

Comment:
I'm actually a little surprised that `gpr_ref_init()` allows you to initialize the refcount to 0.  Are we sure this is okay?

Reply:
Checked `sync.c`, I think it should be okay to initialize the refcount to 0, as long as we won't call `gpr_unref(&ref_)` before calling the first `gpr_ref(&ref_)`.